src/libraries/System.Drawing.Common/src/System/Drawing/Imaging/CachedBitmap.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+using Gdip = System.Drawing.SafeNativeMethods.Gdip;
+
+namespace System.Drawing.Imaging
+{
+    /// <summary>
+    /// Stores a <see cref="Bitmap"/> in a format that is optimized for display on a particular device.
+    /// </summary>
+    public sealed partial class CachedBitmap : IDisposable
+    {
+        internal static bool IsSupported = IsCachedBitmapSupported();
```

Comment:
```suggestion
        internal static readonly bool IsSupported = IsCachedBitmapSupported();
```
to cache the value (and the JIT can treat this as JIT-time constant, especially with tiered compiliation).
See comment above.